packages/frontend/src/model/theory_selector.tsx

```diff
@@ -45,6 +45,8 @@ export function TheorySelectorDialog(
     );
 }
 
+export const [selectedTheory, setSelectedTheory] = createSignal<string | undefined>(undefined);
```

Comment:
I don't know if creating a signal outside of a component, for example at the top level, will work. More importantly, we shouldn't need to create a signal at all here. The selected theory is already available from the `LiveModelDocument`. See below.